### DISCUSSION TITLE
Looking for opensource or paid community platform + Online quiz with leaderboard

### DISCUSSION THREAD

**Discussion by 4Teen7Teen** [community/community#46818] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

A Community platform to host 10 lakh users that allows users to have discussions in :
1. Groups
2. Open forums
3. 1on1 chat
4. Post

Also, have an online quiz or challenge platform (has leaderboard) integrated to the community platform

**Comment by Juanescacha** [comment]
Well you could use a Discord server to have all your comunity forums, chats an discussions organized and you can use any website that provides quiz creation and leaderboards like Socrative, or Kahoot!.
